### ISSUE TITLE
Custom Fonts still don't work

### ISSUE THREAD

**Issue by edcousins** [reduxframework/redux-extensions#214]
Not getting any response on this anymore so logging here. Still getting emails daily from customers who can't use the custom fonts extension.

Dovy is aware of the issue(s) and has promised an updated version for a while.

Need this ASAP.

**Comment by edcousins** [comment]
Another 3 weeks, getting silly now. bump.

**Comment by abouolia** [comment]
+1 :( :(

**Comment by kprovance** [comment]
Guys, here is the truth:  It's broken, it's going to take substantial work to fix, and it doesn't sell, and since Redux in general doesn't pay our bills or feed our families, both Dovy and I have gotten tired of dumping endless hours of work into projects that see little to no return.  It sucks, I know, but that's how it is.  Devs don't want to be supportive of tools in which they make serious bank, but give back almost nothing.  We literally have graphs that show the amount of money theme and plugin devs who use Redux make based on OUR work, and we've posted it before.  The tepid ro non-response has showed us that, hey, they don't care.  One dev's attitude was 'hey, if Redux doesn;t pay, get a second job.  Not my problem."  Nice, huh?

Obviously, I cannot speak for Dovy, but speaking for myself, it EXTREMELY frustrating, and speaking for myself, I'm over putting in tireless hours for...well, just about nothing.  We're posted - time and again - over the years that we need community support, and it appears the the community believes the next guy will someone be that guy.  The buck gets passed and passed and passed, but we never see it. It's a painful lesson we've learned.  It's my understanding the Dovy is going to offer something as a replacement.  He'll have to make that announcement.

That's the reality.  Sorry guys.
